### DISCUSSION TITLE
How to place the script next to the head in nuxt3

### DISCUSSION THREAD

**Discussion by tendecade** [nuxt/nuxt#23265] [Questions]
### Describe the feature

Scenario:
My client needs to place Google's GA code in the Head; Google requires this: "Please copy and paste this code into the code of each webpage on your website, immediately following the<head>element“
But after I made changes to the app ->head ->script in 'nuxt. config. ts', the script code has been running after<meta>, and I don't know how to implement it
I think I need help. Thank you 🙏

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by xjccc** [comment]
set `tagPosition: 'head'`
